packages/lodestar-cli/test/unit/lodestar/account.test.ts

```diff
@@ -0,0 +1,114 @@
+import yargs from "yargs/yargs";
+import { expect } from "chai";
+import { stub, match } from "sinon";
+
+import { rootDir } from "../../constants";
+import { account } from "../../../src/cmds/account";
+import { ValidatorDirManager } from "../../../src/validatorDir";
+import { getAccountPaths } from "../../../src/cmds/account/paths";
+import { init } from "../../../src/cmds/init";
+import { TestnetName } from "../../../src/testnets";
+import { medallaConfig } from "../../../src/testnets/medalla";
+import { existsSync, readFileSync, readdirSync, unlinkSync } from "fs";
+
+describe.only("account cli", function() {
+  this.timeout("10 min");
+
+  const spy = stub(console, 'log');
+
+  const testnetName = "medalla";
+  const initDefaults = {
+    rootDir,
+    preset: "mainnet",
+    testnet: testnetName as TestnetName,
+    paramsFile: `${rootDir}/config.yaml`,
+    params: {
+      "DEPOSIT_CHAIN_ID": 5,
+      "DEPOSIT_NETWORK_ID": 5
+    },
+  }
+
+  const walletsDir = `${rootDir}/wallets/`;
+  const walletName = "primary";
+
+  const accountDefaults = {
+    ...initDefaults,
+    name: walletName,
+    passphraseFile: "primary.pass",
+  };
+
+  it("should init beacon configuration with --testnet option", async function() {
+    await new Promise(resolve => yargs()
+      .default(initDefaults)
+      .command(init).help().parse(["init"], resolve));
+    await new Promise(resolve => setTimeout(resolve, 6000));
```

Comment:
Why does it need a timeout? doesn't the command resolve when done?

Reply:
When `parse` resolves, it only resolves the parsing of `init`.  Meanwhile, `init` itself runs as a separate process, and I can't find a way within yargs to be able to hook into the handler to see when it's done running.  So, the timeout is a way to get around this by estimating how long it will take for the handler to finish resolving.  Admittedly, not the best option, and I'm open to other suggestions if you have any.